pkg/packagekit/updates.scss

```diff
@@ -137,6 +137,11 @@ div.changelog {
     margin: 10ex auto 0;
 }
 
+/* workaround font not supporting tabular numbers yet https://github.com/cockpit-project/cockpit/issues/15090 */
+.pf-c-progress__status {
```

Comment:
At first I thought this needs to be more specific -- but we have two progress bars on the Updates page, and they both show percentages. So it's fine to apply this to both of them.